new/track-rfcs.md

```diff
@@ -0,0 +1,96 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io>
+Status: Draft
+Type: Process
+---
+
+# Tracking for Standards Track RFCs
+
+Standards Track RFCs are intended for implementation.  This proposal aims to
+make it easier to tell when an RFC is implemented or orphaned, or what its
+implementation status is.
+
+## Motivation
+
+    As a Chef Contributor,
+    I want to know whether an RFC is actually being worked on,
+    so that I can decide whether to wait, write it myself, or write a replacement RFC.
+
+    As a Chef Contributor,
+    I want to know whether an RFC has become orphaned,
+    So that I can decide whether to find a new owner for the subject.
+
+## Specification
+
+### Tracking Metadata Field
+
+RFCs will now include a metadata field called Tracking, which will generally be
+a list of URLs to issues or PRs where the feature is tracked, and may also link
```

Comment:
I'd like the RFC author, as the implementor/wrangler, to have an issue rather than a PR. It's easy for the author to edit the issue to update the issue to update the status, e.g. a todo list, a list of related/blocking PRs against other projects, words about priority conflicts. On the other hand, RFCs go through a process, and all the merging should be done by editors. If a PR is closed and another one is opened to replace it, the RFC author can easily update the issue or just accept the link that github automatically adds. This is a lot easier than the RFC author coming back to chef-rfc, making a change, and having it go through an approval process. Even if we got rid of the process and said RFC authors could update the tracking information at will for their RFCs, it's not a set of changes we really need to be tracking it git.